src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -690,4 +690,13 @@
   <data name="ObjectCreationHandlingPropertyCannotAllowReferenceHandling" xml:space="preserve">
     <value>JsonObjectCreationHandling.Populate is incompatible with reference handling.</value>
   </data>
+  <data name="FormatInt128" xml:space="preserve">
```

Comment:
I think you could collapse all these messages to a single string that accepts the type name as a parameter.

Reply:
Yes that is true, but we should do it for all the `Format*` strings.
https://github.com/dotnet/runtime/blob/f7ad7265b222bede49ff93b341d81b61bd898122/src/libraries/System.Text.Json/src/Resources/Strings.resx#L265-L285